src/components/ui/IconButton/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"; 
+import { View } from "react-native";
+import Icon from "@app/components/ui/Icon";
+import { colors } from "@app/utils";
+
+interface IconButtonProps {
+  id?: string;
+  onPress: () => void; 
+}
+
+const IconButton: React.FC<IconButtonProps>  = ({id, onPress}) => {
+
+  return(
+    <View>
+      <Icon 
+        id={`${id}-icon`}
+        name="add-circle"
+        onPress={() => onPress()}
+        color={colors.dark}
```

Comment:
It would be good to add the additional flags for it:

- raised (for the box shadow)
- reverse (to be able to specify the background color)

https://reactnativeelements.com/docs/icon/